controllers/infra/api/v1/infra_types.go

```diff
@@ -91,6 +92,8 @@ type Hosts struct {
 	Metadata []Metadata `json:"metadata,omitempty"`
 	// Find the mapping between expected hosts and actual hosts
 	Index int `json:"index,omitempty"`
+	// Availability Zone
+	AvailabilityZone string `json:"availability_zone,omitempty"`
```

Comment:
Seems all the hosts will in the same zone? If the zone is same, put this failed in `Spec.AvailabilityZone` is better?